config/m4/ib.m4

```diff
@@ -7,6 +7,11 @@
 #
 
 
+AC_ARG_ENABLE(res-domain,
```

Comment:
no need for changing ib.m4, the config can be runtime only

Reply:
removed